processes/release_processes/firecloud-develop.dot

```diff
@@ -71,10 +69,8 @@ digraph {
   confirm_cromwell_v -> fcdev_success
 
   fcdev_success -> jenkins_reset
```

Comment:
unrelated to the spirit of this PR but it seemed like `dsl-seed-job` [ran automatically](https://fc-jenkins.dsp-techops.broadinstitute.org/job/dsl-seed-job/1238/) on `master` the last time I merged my `dsp-jenkins` PR for 68.1, so there may be another bubble that has been automated out of the process

Reply:
This observation holds up.

The job is configured to [poll for changes](https://fc-jenkins.dsp-techops.broadinstitute.org/job/dsl-seed-job/GitHubPollLog/) to `master` on Github:
<img width="1259" alt="Screen Shot 2021-09-09 at 7 55 35 PM" src="https://user-images.githubusercontent.com/1087943/132777322-047ae9e1-ec1c-470e-af11-28482a6933a3.png">

And runs with [a default](https://fc-jenkins.dsp-techops.broadinstitute.org/job/dsl-seed-job/configure) of `master` if no branch parameter is provided:
<img width="1413" alt="Screen Shot 2021-09-09 at 7 55 16 PM" src="https://user-images.githubusercontent.com/1087943/132777310-a81ba4db-9b13-493f-aaa2-e4b42822ed3d.png">